### PR TITLE
NAS-119655 / 23.10 / `auth.set_attribute` to store something that will be returned in `aut…

### PR DESCRIPTION
**Pull request by themylogin**
…h.me`

### CODE REVIEW

**Comment by themylogin** [comment]
time 7:00

**Review by yocalebo — APPROVED** [review]
Nice! Let's wait for @anodos325 review before merge please.

**Review by anodos325 — APPROVED** [review]

**Pull request merged**
